tools/scripts/sync-i18n.ts

```diff
@@ -44,13 +44,20 @@ const syncChallenges = async () => {
         const langContent = await readFile(targetPath, 'utf-8');
         const engLines = englishContent.split('\n');
         const engId = engLines.find(l => l.startsWith('id'));
+        const engSlug = engLines.find(l => l.startsWith('dashedName'));
         const langLines = langContent.split('\n');
         const langId = langLines.find(l => l.startsWith('id'));
+        const langSlug = langLines.find(l => l.startsWith('dashedName'));
         if (engId && langId && engId !== langId) {
```

Comment:
```suggestion
        if (engId && engId !== langId) {
```
We know it exists, because of the condtional above.